### PR TITLE
chore: remove default exports [P5]

### PR DESCRIPTION
**Pull request by layershifter**
#### Description of changes

This PR is a fifth round of `default` exports removal, see https://github.com/microsoft/fluentui/issues/12953#issuecomment-633970215.

### CODE REVIEW

File: packages/fluentui/e2e/.eslintrc.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
   "extends": ["plugin:@fluentui/eslint-plugin/react-northstar"],
-  "root": true
+  "root": true,
+  "rules": {
+    "import/no-default-export": "off"
```
Comment (pompomon) [inline packages/fluentui/e2e/.eslintrc.json:5]: Please why it's "off" for e2e?

Comment (layershifter) [reply packages/fluentui/e2e/.eslintrc.json:5]: `e2e` is not a part of distributed code so it's not important to fix things there.

File: packages/fluentui/react-northstar/src/index.ts [hunk 2]
```diff
@@ -17,304 +23,184 @@ export * from './themes/teams/types';
...
 export * from './components/Button/ButtonGroup';
-export { default as ButtonGroup } from './components/Button/ButtonGroup';
 export * from './components/Button/ButtonContent';
-export { default as ButtonContent } from './components/Button/ButtonContent';
 
 export * from './components/Chat/Chat';
-export { default as Chat } from './components/Chat/Chat';
 export * from './components/Chat/ChatItem';
-export { default as ChatItem } from './components/Chat/ChatItem';
+export * from './components/Chat/ChatItem';
```
Comment (pompomon) [inline packages/fluentui/react-northstar/src/index.ts:49]:
```suggestion
```
Duplicate line

File: packages/fluentui/docs/src/config.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 const config = {
+  repoURL: 'https://github.com/microsoft/fluentui',
```
Comment (pompomon) [inline packages/fluentui/docs/src/config.ts:2]: Should we import it from `packages/fluentui/react-northstar/src/utils/constants.ts` ? So we don't have to change URL in two places

Comment (layershifter) [reply packages/fluentui/docs/src/config.ts:2]: `constants` are no longer reexported under `@fluentui/react-northstar` (I think that this variable was exported by a mistake).

**Pull request merged**